Marlin/src/inc/SanityCheck.h

```diff
@@ -874,6 +874,31 @@ static_assert(Y_MAX_LENGTH >= Y_BED_SIZE, "Movement bounds (Y_MIN_POS, Y_MAX_POS
   #error "INDIVIDUAL_AXIS_HOMING_MENU is incompatible with DELTA kinematics."
 #endif
 
+/**
+ * Multi-Material-Unit requirements
+ */
+#if HAS_PRUSA_MMU2
+  #if EXTRUDERS != 5
+    #undef SINGLENOZZLE
```

Comment:
I think this is senseless code will not compile

Reply:
I put this #undef here to suppress a sanity-check error message that makes reference to "`SINGLENOZZLE`" because the `SINGLENOZZLE` option is only enabled as a condition for the MMU, and the user does not need to enable it themselves.

Reply:
There's a good reason for it, but GitHub ate my last reply and I don't feel like typing it out again.